ansible/roles/tasks-systemd/tasks/main.yml

```diff
@@ -63,18 +63,29 @@
           '--memory=24g',
           '--pids-limit=16384',
           '--shm-size=1024m',
+
           # qcow overlays on tmpfs
           '--tmpfs=/tmp:size=14g',
           '--env=TEST_OVERLAY_DIR=/tmp',
+
+          # image cache
+          '--env=COCKPIT_IMAGES_DATA_DIR=/cache/images',
```

Comment:
I feel like I'm missing the corresponding change in `run-local.sh`

Reply:
It's [right here](https://github.com/cockpit-project/cockpituous/pull/590/commits/fe9946094e4366a2cd55ea833084f674fd89ef97#diff-889d7681af39462e3296875439af2aa5a56e43a0085c273b9e80d84b01b93b32R252), in this very commit.

Reply:
Ah, I didn't understand how this works differently (ie: args given to outside container and then forwarded inside, with job-runner.toml created by shellscript).

I feel like it would be easier to understand (in `run-local.sh`) if the never-changing parts of the run-args are listed directly in the toml file and we only use the variable substitution for the parts we change when we're mocking.